share/github-backup-utils/ghe-restore-mysql

```diff
@@ -43,12 +38,33 @@ is_binary_backup_feature_on(){
   ghe-ssh "$GHE_HOSTNAME" ghe-config --true "mysql.backup.binary"
 }
 
+if ghe-ssh "$GHE_HOSTNAME" test -f /etc/github/cluster ; then
+  ghe_mysql_master=$(ghe-ssh "$GHE_HOSTNAME" ghe-config "cluster.mysql-master")
+  if [ -z $ghe_mysql_master ]; then
+    ghe_mysql_master=$GHE_HOSTNAME
+  else
+    port=$(ssh_port_part "$GHE_HOSTNAME")
+    ghe_mysql_master=$ghe_mysql_master${port:+:$port}
```

Comment:
I'm pretty sure we recommend/require restoring via the MySQL master in Cluster envs, but my memory could be a little rusty here.

Additionally, I don't think you need to do the port determination here. `ghe-ssh` does it: https://github.com/github/backup-utils/blob/178da66f3cbefbb16b0909cc3ed8310f47bfb0d1/share/github-backup-utils/ghe-ssh#L53-L55

Reply:
I did that because `$ghe_mysql_master` doesn't have ports. If a port different from default is specified, `$ghe_mysql_master` won't be able to get it